### ISSUE TITLE
Passing through class to the SelectList component causes error

### ISSUE THREAD

**Issue by robalexclark** [stsrki/Blazorise#620]
Passing through the class to the SelectList component like this:

```cshtml
<SelectList class="form-control" Data="@suppliersList" TextField="@((item)=>item)" ValueField="@((item)=>item)" SelectedValue="@searchModel.Supplier" SelectedValueChanged="@(e => {searchModel.Supplier = e.ToString();} )" />
```

causes an error:

InvalidOperationException: Object of type 'Blazorise.Components.SelectList`1[[System.String, System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]]' does not have a property matching the name 'class'.

Reading through other issues I expected the class name to be passed through (attribute splatting??). Is this a bug or am I doing something wrong?

**Comment by stsrki** [comment]
This is a bug. A forgot to add attribute splatting to the SelectList.

**Comment by robalexclark** [comment]
Thanks for confirming. Are there any plans to release 0.9 as a nuget "prerelease" package?

**Comment by stsrki** [comment]
I have just released 0.9.0-preview2. Here you can find information on how to get it: https://github.com/stsrki/Blazorise#try-preview
